Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/WorldTrackingSceneView.swift

```diff
@@ -192,26 +180,23 @@ public struct WorldScaleGeoTrackingSceneView: View {
         }
     }
     
-    /// If necessary, updates the scene view's camera controller for a new location coming
-    /// from the location datasource.
+    /// Updates the scene view's camera controller with a new location coming from the
+    /// location data source and resets the AR session when using world-tracking configuration.
     /// - Parameter location: The location data source location.
-    @MainActor
-    private func updateSceneView(for location: Location) {
+    private func updateWorldTrackingSceneView(for location: Location) {
         // Do not update the scene view when the coaching overlay is in place.
         guard !coachingOverlayIsActive else { return }
         
-        // Do not use cached location more than 10 seconds old.
-        guard abs(lastLocationTimestamp?.timeIntervalSinceNow ?? 0) < 10 else { return }
+        // Update if the location is more than 10 seconds old.
+        guard abs(lastLocationTimestamp?.timeIntervalSinceNow ?? 0) < timeThreshold else { return }
         
         // Make sure that horizontal and vertical accuracy are valid.
-        guard location.horizontalAccuracy > validAccuracyThreshold,
-              location.verticalAccuracy > validAccuracyThreshold else { return }
+        guard location.horizontalAccuracy > .zero,
```

Comment:
I changed them to a fixed `zero` because the doc says negative value means invalid.

Reply:
So zero would be perfectly precise then? If so, wouldn't we want to update the location in that case?

Reply:
> The accuracy indicates the radius of the location display circle. A negative value indicates that the latitude and longitude are invalid. [doc](https://developer.apple.com/documentation/corelocation/cllocation/1423599-horizontalaccuracy#discussion)

Greater than or equal to `zero` means that when they are negative, we can return early because the location is invalid.

Reply:
> Greater than or equal to zero 

but the check is > not >=

Reply:
Fixed in 2ba0fe3